exploratory_data_analysis/pubtator_exporatory_data_analysis.py

```diff
@@ -0,0 +1,257 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:biovectors]
+#     language: python
+#     name: conda-env-biovectors-py
+# ---
+
+# # Pubtator Central Update
+
+# Pubtator Central updates their data monthly; however, they changed their data to be xml format instead of common text.
+# Based on this update it is imperative to know what changes have been made/format this data to make future experiments easier to work with.
+# This notebook is being created to make training word2vec a whole lot easier.
+
+# +
+from collections import defaultdict, Counter
+import csv
+import itertools
+import lzma
+from pathlib import Path
+import pickle
+import tarfile
+
+import lxml.etree as ET
+import numpy as np
+import pandas as pd
+import plotnine as p9
+import tqdm
+
+from biovectors_modules.word2vec_run_helper import (
+    PubMedSentencesIterator,
+    PubtatorTarIterator,
+    chunks,
+)
+# -
+
+# # Look at xml example
+
+# Lets look at an example entry for each tagged document. Looks like it is regular BioCXML format which is good for processing.
+
+pubtator_abstract_batch = list(Path("../pubtator_abstracts").rglob("*.gz"))
```

Comment:
Sorry if this was in another PR, but maybe adding where this is coming from would be helpful -- channeling Ben's reproducibility talk:)